src/Security/EditableRolesBuilder.php

```diff
@@ -51,6 +51,9 @@ class EditableRolesBuilder
      */
     private $configuration;
 
+    /**
+     * @param array<string, string[]> $rolesHierarchy
```

Comment:
```suggestion
     * @param array<string, array<string>> $rolesHierarchy
```
better support for phpstorm